### ISSUE TITLE
Create testsuite for integration testing

### ISSUE THREAD

**Issue by erdlet** [mvc-toolbox/mvc-toolbox#40]
At the moment we don't have integration tests which e. g. verify the correctness of our JSP tags. TBH this is a very unsatisfying state and we really should add an Arquillian based testsuite.

**Comment by chkal** [comment]
Agreed! I guess we really need integration tests for some of the features.

BTW: I wonder if there is any way to unit-test JSP tags. :-)

**Comment by erdlet** [comment]
In thought about that today too and I think we could provide an alternative `HtmlWriter` which writes into a String instead of `JspContext`. Then we can test the output. When we then find a way to mock the access to `CDI` too, we should be able to test all tags.
